themes-default/slim/test/specs/add-recommended.spec.js

```diff
@@ -0,0 +1,31 @@
+import test from 'ava';
+import Vuex from 'vuex';
+import VueRouter from 'vue-router';
+import { createLocalVue, mount } from '@vue/test-utils';
+import fixtures from '../__fixtures__/app-header';
```

Comment:
What about the fixtures? It's doesn't seem like a good idea to use the `app-header` fixture for all these components.

Reply:
I was thinking we should add a common one?

Reply:
Sounds good...